### ISSUE TITLE
Is there a big difference in the model quality between a loss of 3.0 and 2.9?

### ISSUE THREAD

**Issue by vesuppi** [karpathy/nanoGPT#251]
I am training on a single GPU, and the loss gets ~3.1 after ~24 hours of training, it looks like now the loss is decreasing very slowly. Just out of curiosity, is there a big difference in the model quality between a loss of 3.1, 3.0 and 2.9? Thanks! Training for another few days may reduce the loss down to ~2.9 (haven't tried yet). Is it worth it?

**Comment by karpathy** [comment]
Yes, big difference for any additional loss, the farther and farther you get in training.

**Comment by karpathy** [comment]
The first few amounts of loss are just the most boring things, like learning that sentences end with ".", and that spaces are important. All the interesting stuff gets learned much later and only yields a tiny amount of loss relatively speaking.

**Comment by vesuppi** [comment]
Ah I see, thank you very much!

**Comment by TahaBinhuraib** [comment]
@vesuppi which GPU are you using? Also, could you share your loss curves? I'm trying to pretrain on a single RTX 4090.
